### PR TITLE
Fix: missing event_type field

### LINKED ISSUE #939: Missing a required field in the telemetry for Ping-Centre
One of our Ping-Centre ETL jobs has reported that there is a missing required field "event_type" in the payload. See the payload schema [here](https://github.com/mozilla/ping-centre/blob/master/src/schemas/testpilot.js#L7).

As a result, all pings won't be loaded into the database if there is any missing required field. Please fix this issue by adding this field to the payload. Also, we can run an ad hoc job to reload the previous pings if using a dummy "event_type" is acceptable.

### PR DESCRIPTION
**Pull request by ericawright**
r=@bwinton
r=@ncloudioj
r=@clouserw

Add mandatory field event_type and rewrite metrics.md

I'm not entirely certain that the current state of the metrics is acceptable, if I could get a general look over that would be fantastic.

Fixes #939.

### CODE REVIEW

File: docs/metrics.md [hunk 1]
```diff
@@ -129,31 +129,54 @@ An example payload (within the full Telemetry ping):
...
+    {"version",                         "INTEGER",    nil,    nil,        "version"},
+    {"timestamp",                       "TIMESTAMP",  nil,    "SORTKEY",  "Timestamp"},
+    {"uuid",                            "VARCHAR",    36,     nil,        get_uuid},
+    {"service",                         "VARCHAR",    255,    nil,        "test"},
+    {"tour_completed",                  "BOOLEAN",    nil,    nil,        "tour_completed"},
+    {"tab_center_tabs_on_top",          "BOOLEAN",    nil,    nil,        "tab_center_tabs_on_top"},
+    {"tab_center_show_thumbnails",      "BOOLEAN",    nil,    nil,        "tab_center_show_thumbnails"},
+    {"tab_center_window_id",            "INTEGER",    nil,    nil,        "tab_center_window_id"},
+    {"tab_center_currently_toggled_on", "BOOLEAN",    nil,    nil,        "tab_center_currently_toggled_on"},
+    {"tour_completed",                  "BOOLEAN",    nil,    nil,        "tour_completed"},
```
Comment (wresuolc) [inline docs/metrics.md]: this is in here twice

**Comment by wresuolc** [comment]
How can the schema change so much without updating the code?  or is the schema getting updated to match the code?

Either way, /cc @sunahsuh

**Comment by ericawright** [comment]
@wresuolc the small change in `utils.js` makes a big difference. And a portion of it is being updated to match the code.

**Comment by sunahsuh** [comment]
@ilanasegall i think this ETL is still running under your purview -- let me know if you'd like me to take over on that

**Comment by wresuolc** [comment]
I'm happy if @sunahsuh and @ilanasegall are happy

**Pull request merged**